source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -122,11 +122,13 @@ content: |
         :tabid: java-sync
 
         .. code-block:: java
-           :emphasize-lines: 2
+           :emphasize-lines: 3
 
-           final Map<String, Object> extraOptions = new HashMap<String, Object>() {{
-             put("mongocryptdSpawnPath", "/usr/local/bin/mongocryptd");
-           }};
+           final Map<String, Object> extraOptions = new HashMap<String, Object>() {
```

Comment:
We use the "double curly brace pattern" for inline declaration of data in a ``HashMap``, which is formatted as such rather than separating the lines in other files in the mongo-java-driver repo such as: https://github.com/mongodb/mongo-java-driver/blob/master/driver-sync/src/examples/tour/ClientSideEncryptionAutoEncryptionSettingsTour.java#L56

Until we have some formatting standard that addresses this specific case, I think we should continue to use the pattern. What do you think?

Reply:
I'm 'pro' it if it fits the convention that's already set. Just made that change, let me know what you think.